src/impls.rs

```diff
@@ -618,27 +609,24 @@ impl<T: Config> Pallet<T> {
 	/// The number of values submitted by the given reporter to the given query identifier.
 	pub fn get_reports_submitted_by_address_and_query_id(
 		reporter: AccountIdOf<T>,
-		query_id: QueryIdOf<T>,
+		query_id: QueryId,
 	) -> u128 {
 		<StakerDetails<T>>::get(reporter)
 			.and_then(|stake_info| stake_info.reports_submitted_by_query_id.get(&query_id).copied())
 			.unwrap_or_default()
 	}
 
 	pub(super) fn _get_reward_amount(
-		feed_id: FeedIdOf<T>,
-		query_id: QueryIdOf<T>,
-		timestamp: TimestampOf<T>,
+		feed_id: FeedId,
+		query_id: QueryId,
+		timestamp: Timestamp,
 	) -> Result<AmountOf<T>, Error<T>> {
-		ensure!(
-			T::Time::now().saturating_sub(timestamp) < T::ClaimPeriod::get(),
-			Error::<T>::ClaimPeriodExpired
-		);
+		ensure!(Self::now().saturating_sub(timestamp) < 4 * WEEKS, Error::<T>::ClaimPeriodExpired);
```

Comment:
@tkernell Can you verify this?

Reply:
Relates to here: https://github.com/tellor-io/autoPay/blob/ffff033170db06e231fba90213db59b4dc42b982/contracts/Autopay.sol#L712

Reply:
Yes, autopay feed rewards must be claimed with 4 weeks of data submission